appserver/tests/payara-samples/samples/asadmin/src/test/java/fish/payara/samples/asadmin/SetHazelcastConfigurationTest.java

```diff
@@ -39,15 +39,22 @@
  */
 package fish.payara.samples.asadmin;
 
+import fish.payara.nucleus.hazelcast.HazelcastRuntimeConfiguration;
+import fish.payara.samples.Unstable;
+
 import org.glassfish.embeddable.CommandResult;
 import org.junit.Before;
 import org.junit.Test;
-
-import fish.payara.nucleus.hazelcast.HazelcastRuntimeConfiguration;
+import org.junit.experimental.categories.Category;
 
 /**
  * Verifies the correctness of the {@code SetHazelcastConfiguration} command.
  */
+@Category(Unstable.class)
+// Fails from two reasons:
+// 1) Requires completely new domain. Side effects of other tests break this one.
+// 2) On JDK8 fails because of usage of @Category annotation which has problems with
```

Comment:
Why are we introducing things that don't work on what is arguably the primary java version?

Reply:
Agree. Eventually we might start using JUnit5 but this should be a planned step.

Reply:
First Categories are used in JUnit4 as a standard for maybe 15 years and they do work. 

Probably this issue is caused by interaction with this in PayaraArquillianTestRunner. But it is worthless to investigate, because the test does not work at all.
And if it would, we would not need the category ;)

![image](https://user-images.githubusercontent.com/302904/81013922-41c84200-8e5c-11ea-9c4c-a46684410146.png)

Reply:
https://dzone.com/articles/closer-look-junit-categories (2010)

Reply:
Okay, thought the issue here was related to use of a feature that was introduced later.

Reply:
Maybe we should use categories instead of NotMicroCompatible too, but ... I think there will be more work with Andrew's point around repeatibility of tests. Probably it could spark some more stable (and faster and cheaper) architecture of tests, I will think about it :-)